cmd/controller-manager/app/controller-manager.go

```diff
@@ -98,6 +99,8 @@ member clusters and do the necessary reconciliation`,
 	flags.StringVar(&kubeFedConfig, "kubefed-config", "", "Path to a KubeFedConfig yaml file. Test only.")
 	flags.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
 	flags.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+	flags.IntVar(&restConfigQPS, "rest-config-qps", 5, "QPS of rest config.")
```

Comment:
Just to confirm, are these the default values ?

Reply:
yes, they are default values.

Reply:
1